content/en/docs/making-requests.md

```diff
@@ -426,6 +426,195 @@ functional arguments to `NextPage`. For more information about overriding client
+When interacting with AWS APIs that are asynchronous, you often need to wait 
+for a particular resource to become available in order to perform further 
+actions on it. 
+
+Let’s take an example of {{% alias service=DDBlong %}} `CreateTable` API. 
+After you invoke the API, the response returns immediately with a TableStatus 
+of CREATING, and you can’t invoke read or write operations until the table 
+status has been transitioned to `ACTIVE`. 
+
+Writing logic to continuously poll the table status can be cumbersome 
+and error-prone. This is especially true when you are dealing with multiple 
+AWS resources, as different resources often have different polling APIs and 
+success states that indicate the availability of that particular resource
+
+The waiters help take the complexity out of it and are simple APIs that handle 
+the polling task for you. It also provides polling configurations, such as the 
+maximum wait time, minimum delay and maximum delay between polling retries. 
+Waiters use a back-off strategy between each attempt. With the waiters utility, 
+you can focus on working with the resource without worrying how to make sure 
+that this resource is ready.
+
+The following example shows how you can use waiters to poll if a 
+{{% alias service=DDB %}} table is created and ready for a write operation.
+
+```go
+import "context"
+import "fmt"
+import "log"
+import "time"
+import "github.com/aws/aws-sdk-go-v2/aws"
+import "github.com/aws/aws-sdk-go-v2/config"
+import "github.com/aws/aws-sdk-go-v2/service/dynamodb"
+
+// ...
+
+cfg, err := config.LoadDefaultConfig(context.TODO())
+if err != nil {
+    log.Printf("error: %v", err)
+    return
+}
+
+client := dynamodb.NewFromConfig(cfg)
+
+// we create a waiter instance by directly passing in a client
+// that satisfies the waiters client Interface. 
+waiter :=  dynamodb.NewTableExistsWaiter(client)
+
+// params is the input to api operation used by the waiter
+params := &dynamodb.DescribeTableInput {
+	TableName: aws.String("test-table")
+}
+
+// maxWaitTime is the maximum wait time, the waiter will wait for 
+// the resource status.
+maxWaitTime := 5 * time.Minutes
+
+// Wait will poll until it gets the resource status, or max wait time 
+// expires.
+err := waiter.Wait(context.TODO(), params, maxWaitTime)  
+if err == nil {
+    log.Printf("error: %v", err)
+    return 
+}
+fmt.Println("Dynamodb table is now ready for write operations")
+
+```
+
+Since the waiter constructor takes in a client interface, you can 
+provide a custom client that will be used by the waiter to poll the resource.
+This design also makes it easier to mock the waiter for testing purposes,
+
+#### Overriding waiter configuration
+
+By default, SDK uses the minimum delay and maximum delay value configured with 
+optimal values defined by AWS services for different APIs. You can override these
+configurations on the entire waiter or per request, via waiter-specific 
+functional options present for each waiter. 
```

Comment:
You can override waiter configuration by providing functional options during waiter construction, or when invoking a waiter operation.